CX-SDK-OIDC-login/src/main/java/com/cx/sdk/oidcLogin/webBrowsing/OIDCWebBrowser.java

```diff
@@ -178,13 +178,24 @@ private static void close() {
     }
 
     public static Engine defaultEngine() {
+
         if (ENGINE == null || ENGINE.isClosed()) {
-            ENGINE = Engine.newInstance(EngineOptions
-                    .newBuilder(RenderingMode.HARDWARE_ACCELERATED)
-                    .userAgent("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36")
-                    .addSwitch("--disable-google-traffic")
-                    .licenseKey(JxBrowserLicense.getLicense())
-                    .build());
+            if(System.getProperty("RenderingMode")!=null && !System.getProperty("RenderingMode").isEmpty() && System.getProperty("RenderingMode").equalsIgnoreCase("OFF_SCREEN")) {
```

Comment:
Instead of using generic name for property lets use something specific like JxBrowserEngineRenderingMode

Reply:
OK sure